code/modules/dwarven/dwarven_items.dm

```diff
@@ -0,0 +1,244 @@
+/obj/item/book_of_babel/dwarven_guide
+	name = "Brokering 101: Dwarven guide"
+	desc = "An ancient tome written dwarven"
+	icon = 'icons/obj/library.dmi'
+	icon_state = "book1"
+	w_class = 2
+
+/obj/item/book_of_babel/dwarven_guide/attack_self(mob/living/carbon/human/user)
+	if(!user.has_language(/datum/language/dwarven))
+		return FALSE
+	. = ..()
```

Comment:
This grants all languages in parent? Should only grant dwarven.

Reply:
this grants all langs since no dwarves speak common, instead one of the dwarves is a dedicated broker that can speak all langs

Reply:
Oh. I assumed it taught visitors dwarven, gotcha.